pypeit/par/pypeitpar.py

```diff
@@ -1726,6 +1726,10 @@ def __init__(self, obj_toler=None, assign_obj=None, nsig_thrshd=None,
         dtypes['assign_obj'] = bool
         descr['assign_obj'] = 'If SlitMask object was generated, assign RA,DEC,name to detected objects'
 
+        defaults['skip_serendip'] = False
+        dtypes['skip_serendip'] = bool
+        descr['skip_serendip'] = 'Skip the extraction of non detected (i.e, serendipitous) objects.'
```

Comment:
Does it mask them but not extract them or does it ignore them altogether?

Reply:
Why would we not want to extract them? Just to save time? Also, local sky-subtraction will not work if the serendips are not in there, in other words if a faint object is near a bright star, you cannot skip the serendip bright star. Let's talk through this one.

Reply:
As I said before, it makes zero sense to skip-serendip in local. Even if you mask them in global via the sky mask, they still need to be modeled in local if they are near an object of interest. If you skip them, that is going to produce bad results in some cases. I think the user has the option to increase the  sigma_thresh and only find bright object, but I don't like this skip_serendip option and think it should be removed.